wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx

```diff
@@ -1,39 +1,45 @@
+/* eslint-disable max-lines */
```

Comment:
probably not a part of this PR but we need to restructure to multiple components CreateEntityModal too @terryli0095